provisioner.go

```diff
@@ -342,20 +352,34 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 				{
 					Name:    "local-path-" + string(action),
 					Image:   p.helperImage,
-					Command: append(cmdsForPath, filepath.Join("/data/", volumeDir)),
+					Command: cmdsForPath,
+					Args:    []string{filepath.Join(parentDir, volumeDir), strconv.FormatInt(sizeInBytes, 10)},
 					VolumeMounts: []v1.VolumeMount{
 						{
 							Name:      "data",
 							ReadOnly:  false,
-							MountPath: "/data/",
+							MountPath: parentDir,
 						},
 						{
 							Name:      "script",
 							ReadOnly:  false,
 							MountPath: "/script",
 						},
+						{
+							Name:      "xfs-quota-projects",
+							SubPath:   "projects",
+							MountPath: "/etc/projects",
+						},
+						{
+							Name:      "xfs-quota-projects",
+							SubPath:   "projid",
+							MountPath: "/etc/projid",
+						},
 					},
 					ImagePullPolicy: v1.PullIfNotPresent,
+					SecurityContext: &v1.SecurityContext{
+						Privileged: &enablePrivileged,
```

Comment:
Because we need to access `/etc`? Seems unnecessary if the user is not running XFS.